Code/Framework/AzCore/AzCore/Component/ComponentApplication.h

```diff
@@ -209,7 +209,7 @@ namespace AZ
         bool DeleteEntity(const EntityId& id) override;
         Entity* FindEntity(const EntityId& id) override;
         AZStd::string GetEntityName(const EntityId& id) override;
-        bool SetEntityName(const EntityId& id, const AZStd::string_view name) override;
+        bool SetEntityName(const EntityId& id, AZStd::string_view name) override;
```

Comment:
update: It is worth keeping this change as it's actually fixing something that's not correct (the `const` being meaningless in this case)